src/components/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+export interface TodoListType {
+  todos: Todo[];
+  inputRef: React.RefObject<HTMLInputElement>;
+}
+
+export const TodoList: React.FC<TodoListType> = ({ todos, inputRef }) => {
+  return (
+    <div>
```

Comment:
Consider using `<ul>` tag here because you have list of todos

Reply:
I saw the code in example and there was div. I will change it thanks for an advice